.env.sample

```diff
@@ -26,4 +26,8 @@ ML_PROJECT_SERVICE_URL = "http://ml-project-service:3000"
 
 KEYCLOAK_PUBLIC_KEY_PATH = "keycloak-public-keys"                               // Keycloak public keys path 
 
-DISABLE_LEARNER_SERVICE_ON_OFF = "ON"                                          // Disable learner service check
\ No newline at end of file
+DISABLE_LEARNER_SERVICE_ON_OFF = "ON"                                          // Disable learner service check
+
+DOCK_URL = "https://dock.sunbirded.org"
```

Comment:
@aishwa8141 Add comments along side with the explanation about what this key is used for